src/pages/index.jsx

```diff
@@ -1,18 +1,23 @@
 import React from 'react';
+import 'bootstrap/dist/css/bootstrap.css';
```

Comment:
Using bootstrap is generally ok but is overkill for this project. you could get the same results using simple css without the need of loading a framework